test-proxy/pom.xml

```diff
@@ -0,0 +1,126 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-bigtable-test-proxy</artifactId>
+  <version>1.0</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Bigtable Test Proxy</name>
+  <url>https://github.com/googleapis/java-bigtable</url>
+
```

Comment:
This definitely needs a description

Reply:
@igorbernstein2 @mutianf are we planning on publishing this module?

Reply:
I think so, is there any concern with adding this in public repo?

Reply:
I dont see any reason not to

Reply:
To maven central? I told Kristen that we're not going to 😅

Reply:
Chatted offline, decided to not publishing it for now.